tests/e2e/system_test/test_cluster_full_and_recovery.py

```diff
@@ -0,0 +1,348 @@
+            expected_used_capacity=85.0,
+        )
+        if not sample.wait_for_func_status(result=True):
+            log.error("The after 18000 seconds the used capacity smaller than 85%")
+            raise TimeoutExpiredError
+
+        log.info(
+            "Verify Alerts are seen 'CephClusterCriticallyFull' and 'CephOSDNearFull'"
+        )
+        log.info("Verify used capacity bigger than 85%")
+        expected_alerts = ["CephClusterCriticallyFull", "CephOSDNearFull"]
+        sample = TimeoutSampler(
+            timeout=600,
+            sleep=50,
+            func=self.verify_alerts_via_prometheus,
+            expected_alerts=expected_alerts,
+        )
+        if not sample.wait_for_func_status(result=True):
+            log.error(f"The alerts {expected_alerts} do not exist after 600 sec")
+            raise TimeoutExpiredError
+
+        for pod_name in ("mon", "mgr", "osd"):
+            log.info(f"Respin pod {pod_name}")
+            disruption = disruption_helpers.Disruptions()
+            disruption.set_resource(resource=f"{pod_name}")
+            disruption.delete_resource()
+
+        log.info("Verify PVC2 [CEPH-FS + CEPH-RBD] are in Pending state")
+        pvc_obj_blk2 = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_PENDING,
+        )
+        pvc_obj_fs2 = pvc_factory(
+            interface=constants.CEPHFILESYSTEM,
+            project=self.project_obj,
+            size=2,
+            status=constants.STATUS_PENDING,
+        )
+        log.info(
+            "Waiting 20 sec to verify PVC2 [CEPH-FS + CEPH-RBD] are in Pending state."
+        )
+        time.sleep(20)
+        helpers.wait_for_resource_state(
+            resource=pvc_obj_blk2, state=constants.STATUS_PENDING
+        )
+        helpers.wait_for_resource_state(
+            resource=pvc_obj_fs2, state=constants.STATUS_PENDING
+        )
+
+        log.info("Create snapshot from PVC1 and verify snapshots on false state")
+        snap_blk1_obj = snapshot_factory(pvc_obj_blk1, wait=False)
+        snap_fs1_obj = snapshot_factory(pvc_obj_fs1, wait=False)
+        time.sleep(20)
+        snap_blk1_obj.ocp.wait_for_resource(
+            condition="false",
+            resource_name=snap_blk1_obj.name,
+            column=constants.STATUS_READYTOUSE,
+            timeout=60,
+        )
+        snap_fs1_obj.ocp.wait_for_resource(
+            condition="false",
+            resource_name=snap_fs1_obj.name,
+            column=constants.STATUS_READYTOUSE,
+            timeout=60,
+        )
+
+        log.info("Change Ceph full_ratiofrom from 85% to 95%")
+        change_ceph_full_ratio(95)
+
+        log.info("Delete  benchmark-operator PVCs")
+        self.benchmark_obj.cleanup()
+        self.banchmark_operator_teardown = False
+
+        log.info("Change Ceph full_ratio from from 85% to 95%")
```

Comment:
Please correct the log message, you are changing the backfill full ratio

Reply:
done